lib/model/store.dart

```diff
@@ -420,6 +429,7 @@ class PerAccountStore extends ChangeNotifier with EmojiStore, UserStore, Channel
   final GlobalStore _globalStore;
   final ApiConnection connection; // TODO(#135): update zulipFeatureLevel with events
 
+  String queueId;
```

Comment:
```suggestion
  final String queueId;
```

Right? (Just like it was when on UpdateMachine.)